lib/cdo/web_purify.rb

```diff
@@ -1,5 +1,6 @@
 require 'open-uri'
 require 'json'
+require_relative '../../pegasus/src/env'
```

Comment:
This is needed for `CDO`.

Reply:
Unfortunately this is causing problems with the `ApiControllerTest`, where it attempts to override the [slog](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/controllers/api_controller_test.rb#L5) method :( We have encountered this before. It seems not entirely deterministic, as it passed in circle and one DTT, but now has failed DTT a couple times. I'll look into a better way for that test to run without relying so much on dependency order, but for now reverting this until we figure out a fix.

Reply:
@islemaster I don't know. Since it appears to be just for `CDO` then it's probably unnecessary and `../../deployment` is fine.

In semi-related news, I [fixed](https://github.com/code-dot-org/code-dot-org/pull/18164) the above-mentioned errors from overriding the `slog` method last year. I don't think I updated this PR with that info.

Reply:
Oh, this PR was reverted and re-introduced in https://github.com/code-dot-org/code-dot-org/pull/18174, with comments about the fix.

Reply:
We're considering cutting this thread (it might be the lone dashboard -> pegasus dependency in our production environment) after we clean up some tech debt with i18n.  Just wanted to see if there was some particular reasoning behind this we were missing.  We also need to be careful - who knows what we've built that depends on this subtle connection over the last seven months.

Reply:
I'm all for it. Conceptually, the root `lib` dir should not depend on anything in either dashboard or pegasus.